Automattic/wp-calypso
# Domain management: Inline status notice

Related to p58i-fm0-p2#comment-59166.

## Proposed Changes

| Desktop | Mobile |
| --------- | ------ |
| <img width="1099" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/133287f2-e073-4480-b13e-5576e01e8862"> | <img width="478" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/1fead7fe-caa5-4d41-8284-7a82502cee34"> |

## Testing Instructions

Check that the CTAs are showing to the right of the statuses. As a more comprehensive testing method, check the unit tests. They should all pass.
